crates/node/core/src/args/engine.rs

```diff
@@ -17,9 +17,16 @@ pub struct EngineArgs {
     pub memory_block_buffer_target: u64,
 
     /// Enable state root task
+    ///
+    /// DEPRECATED: state root task is default now, use --engine.legacy-state-root
+    /// to enable the legacy functionality
     #[arg(long = "engine.state-root-task")]
     pub state_root_task_enabled: bool,
```

Comment:
did we ever ship this?

if not we should just remove this entirely

Reply:
good point, it is not present in 1.1.5, removed